packages/components/src/RBACProvider/RBACProvider.tsx

```diff
@@ -38,9 +38,7 @@ export const RBACProvider: React.FunctionComponent<RBACProviderProps> = ({ appNa
   };
 
   useEffect(() => {
-    if (appName) {
```

Comment:
This would trigger the call to fetch permissions twice. It could be better to allow `null` value to be passed as `appName` and if such value request empty application.

Reply:
Yeah, this makes sense. Added `null` support

Reply:
I would flip it arround, if `null` or `string` fire a `fetchPermissions`

Reply:
@karelhala, implemented